patches/react-native-reanimated+3.5.2.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/node_modules/react-native-reanimated/lib/module/reanimated2/index.js b/node_modules/react-native-reanimated/lib/module/reanimated2/index.js
```

Comment:
Why not we fix this import issue in upstream?

Reply:
It was mistake in release 3.5.1 and 3.5.2 and reanimated developers are fully aware of the issue at this point. 
To my knowledge from talking with the maintainer this code was autogenerated in the npm package when it shouldn't and it will be fixed in 3.5.3

Reply:
oh ok, @roryabraham should we wait a bit more until 3.5.3 is released?

Anyway, I confirmed that iOS release build doesn't crash on 3.5.2

Reply:
Yeah, if we include the patch now we'll end up having to do another PR just like this one to bump Reanimated again and get rid of it. Since this isn't urgent, let's wait for `3.5.3`

Reply:
Thanks for updating @wojtus7. @situchan let's resume review here.